flaml/ml.py

```diff
@@ -586,6 +601,7 @@ def compute_estimator(
             groups_val,
             eval_metric,
             task,
+            labels=fit_kwargs.get("label_list"),

```

Comment:
Is this necessary? Is this behavior documented? why is it not in the CV code path?